top_secret.gemspec

```
@@ -31,7 +31,7 @@ Gem::Specification.new do |spec|
   spec.executables = spec.files.grep(%r{\Aexe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "activesupport", "~> 8.0", ">= 8.0.2"
+  spec.add_dependency "activesupport", ">= 7.0.8"
```

I'm wondering if we should do

```suggestion
  spec.add_dependency "activesupport", "~> 7.0", ">= 7.0.8"
```

but also this produced some changes on the Gemfile.lock downgrading the activesupport versions ([here](https://github.com/thoughtbot/top_secret/commit/d2316bc87a93e024726f7398829ba493c34477ba)). Maybe that's what we would want?